src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,14 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
- *
+ * @version TBD
  */
```

Comment:
:no_entry_sign: Missing @package tag in file comment

<sub>hash: 9ab9f078f4103409c69a479dc2c4a966</sub>